middleware/integration_test/docker-compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3"
+services:
+  bitcoind:
+    image: nicolasdorier/docker-bitcoin:0.17.0
+    networks:
+      local:
+        ipv4_address: 10.10.0.10
+    environment:
+      EXPOSE_TCP: "true"
+      BITCOIN_EXTRA_ARGS: |
+        regtest=1
+        whitelist=0.0.0.0/0
+        server=1
+        regtest.rpcallowip=10.211.0.0/16
+        regtest.rpcallowip=172.17.0.0/16
+        regtest.rpcallowip=192.168.0.0/16
+        regtest.rpcallowip=0.0.0.0/16
+        regtest.rpcbind=0.0.0.0
+        rpcport=18443
+        rpcuser=rpcuser
+        rpcpassword=rpcpass
+    expose:
+      - "18443"
+      - "18444"
+    ports:
+      - "0.0.0.0:18443:18443"
+    volumes:
+      - "./volumes/bitcoin:/data"
```

Comment:
You are defining docker managed shared volumes at the bottom of your file 
```
volumes:
  bitcoin_datadir:
  clightning1_bitcoin_datadir:
  clightning2_bitcoin_datadir:
```
but then you're not using them and mounting your volumes with a relative path

The question is, do you need access to the mounted files on the host or just shared between the containers?
If on the host then you can keep the relative path mounts and get rid of the shared volume declarations on the bottom, 

otherwise, you can mount the shared volumes here like:
 ```
volumes:
      - "bitcoin_datadir:/data"
```
without specifying the path